controller/config.go

```diff
@@ -212,6 +216,9 @@ var (
 	// AllowedUpdateConfigAttributes contains all of the controller
 	// config attributes that are allowed to be updated after the
 	// controller has been created.
+	//
+	// TODO: (hml) 2018-07-25
+	// should CloudCACertsKey be in this list?
```

Comment:
Who would know the answer to this question?

Reply:
On reflection, I believe the answer is yes, CloudCACertsKey should be in AllowedUpdateConfigAttributes.  Certificates can expire and need to be refreshed.